Examples/README.md

```diff
@@ -0,0 +1,17 @@
+# Examples
+
+These two examples projects illustrate how VelocidiSDK can be used in a project. ObjcExample is written in Objective-C and SwiftExample is written in Swift.
+
+## Instructions
+
+Setup should be really simple, all that is needed is to open the project with Xcode. After that, run `pod install` and it should be ready to go.
+
+### Capture HTTP events
+
+To see the HTTP events sent by the SDK, have the configuration url's point to `http://localhost:8080` and then run a netcat instance that replies to all request with a `200 OK` response code.
+
+```
+$ while true; do echo -e 'HTTP/1.1 200 OK\r\n\r\n' | nc -i 1 -l 8080; done
+```
+
+_Netcat has limited functionality and this command will wait 1 second before replying(in order to wait for all data to be transmitted ), so the it might feel a little slugish. You can also use online mock HTTP clients, like [Mocky](https://www.mocky.io/)_ 
```

Comment:
I feel like this is saying the same thing just in a different way.

Reply:
Yes, it is. It is a nitpick of mine (I don't like "it might feel a little slugish"). I don't have strong feelings. However, if you ignore, please fix spacing and spelling:

```suggestion
_Netcat has limited functionality and this command will wait 1 second before replying (in order to wait for all data to be transmitted), so it might feel a little sluggish. You can also use online mock HTTP clients, like [Mocky](https://www.mocky.io/)._ 
```